/Users/mac/BlockchainProjects/smartBetting/ethereum/contracts/Betting.sol
node:assert:399
    throw err;
    ^

AssertionError [ERR_ASSERTION]: Invalid callback object specified.
    at runWithCallbacks      (/Users/mac/BlockchainProjects/smartBetting/node_modules/solc/wrapper.js:117:34)
at compileStandard (/Users/mac/BlockchainProjects/smartBetting/node_modules/solc/wrapper.js:223:20)
at Object.compileStandardWrapper [as compile] (/Users/mac/BlockchainProjects/smartBetting/node_modules/solc/wrapper.js:229:20)
at Object.<anonymous> (/Users/mac/BlockchainProjects/smartBetting/ethereum/compile.js:13:21)
at Module._compile (node:internal/modules/cjs/loader:1103:14)
at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
at Module.load (node:internal/modules/cjs/loader:981:32)
at Function.Module._load (node:internal/modules/cjs/loader:822:12)
at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
at node:internal/main/run_main_module:17:47 {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
